packages/kbn-object-versioning/lib/object_transform.test.ts

```diff
@@ -127,7 +129,12 @@ describe('object transform', () => {
 
   describe('down()', () => {
     test('it should down transform to a previous version', () => {
-      const fooTransforms = setup(1);
+      const fooTransforms = setup<
```

Comment:
I added the generic to validate that `fooTransforms.down` casts correctly the `value` returned